yarn.lock

```diff
@@ -2,13 +2,64 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.0.0-beta.51":
+"@babel/cli@^7.0.0-beta.51":
```

Comment:
I have no idea why this is showing in the diff, https://github.com/reactjs/react-transition-group/blob/c3b7708a536a7625770c5131137ba8ec3aea82a2/yarn.lock#L5-L18 is exactly this.